scripts/index.js

```diff
@@ -1,13 +1,9 @@
 // add your solutions here
 function isPositive(nums){
-    result = []
-    nums.forEach(function (num){
-        if (num > 0){
-            result.push(num)
-        }
-    })
-    return result
+    results = []
+    const isPositive = (num) => num > 0;
+    results.push(nums.filter(isPositive))
```

Comment:
One thing though: do you need the `results` array? Check the `results.length` to get an idea of why it might be extraneous.